src/site/content/en/live/index.njk

```diff
@@ -149,6 +149,4 @@ date: 2020-06-31
 
 </div>
 
-<code id="event-data" style="display: none;">
-  {{ event.days | dump | safe }}
-</code>
+<web-event-store style="display: none;">{{ event.days | dump | safe }}</web-event-store>
```

Comment:
It looks like Ewa's maps widget need data form event.json as well. I think it might be good to merge that entire object into the store and not just event.days.